src/robusta/runner/config_loader.py

```diff
@@ -177,6 +187,22 @@ def __reload_playbook_packages(self, change_name):
                 runner_config.playbook_repos[
                     "robusta.core.playbooks.internal"
                 ] = PlaybookRepo(url=INTERNAL_PLAYBOOKS_ROOT, pip_install=False)
+                # order matters! Loading the default first, allows overriding it if adding package with the same name
```

Comment:
I think repositories defined in `active_playbooks.yaml` will be loaded first.

Maybe it's also best to use a list here and not a dictionary. In recent versions of Python, dicts remember the order of insertion, but it's maybe better to be explicit about the order and not rely on that.

Reply:
The iteration order is by insertion order since 3.7
Added a comment